tests/test_DataFrameWrapper.py

```diff
@@ -0,0 +1,211 @@
+from chispa.dataframe_comparer import assert_df_equality
+from pyspark.sql import SparkSession
+from pyspark.sql import DataFrame as SparkDataFrame
+from pyspark.sql.functions import col
+from pyspark.sql.types import StructType, StructField, StringType
+from pytest import fixture
+from pandas import DataFrame
+
+from sds_data_model.dataframe import DataFrameWrapper
+
+# Create dataframe to test against
+expected_data = {'a': [1, 2, 3],
+                 'b': [3, 4, 5],
+                 'c': [5, 6, 7]}
+expected_df = DataFrame(expected_data)
+
+
+@fixture
+def spark_session() -> SparkSession:
+    """Local Spark context."""
+    return (
+        SparkSession.builder.master(
+            "local",
+        )
+        .appName(
+            "Test context",
+        )
+        .getOrCreate()
+    )
+
+
+@fixture(scope="session")
+def temp_file(tmpdir_factory):
+    """Create a temporary directory and data for testing.
+
+    Args:
+        tmpdir_factory (Instance): temporary directory instance
+
+    Returns:
+        object: object containing path to temporary data
+    """
+    p = tmpdir_factory.mktemp("data").join("temp.csv")
+    expected_df.to_csv(str(p), index=False, header=True)
+    return p
```

Comment:
- The `pytest` docs recommend `tmp_path` over `tmpdir` and `tmpdir_factory`, which is my fault as I suggested `tmpdir`.
- You might as well return a string here as you're going to need to convert it to use it
- I don't think test functions need a full docstring, just a oneliner.
- Also, I prefer `temp_path` because your'e returning a path not a file

```suggestion
@fixture
def temp_path(tmp_path, expected_dataframe: SparkDataFrame) -> str:
    """Create a temporary directory and data for testing."""
    path = str(tmp_path / "test.csv")
    expected_dataframe.write.csv(
        path=path,
        header=True,
    )
    return path
```